Plugins/BCClient/Source/BCClientPlugin/Private/BrainCloudWrapper.cpp

```diff
@@ -302,6 +309,12 @@ void UBrainCloudWrapper::smartSwitchAuthenticateUniversal(const FString &userid,
     getIdentitiesCallback(smartCallback);
 }
 
+void UBrainCloudWrapper::smartSwitchAuthenticateAdvanced(EBCAuthType in_authenticationType, const FAuthenticationIds& in_ids, bool in_forceCreate, const FString& in_extraJson, IServerCallback* in_callback)
+{
+    SmartSwitchAuthenticateCallback *smartCallback = new SmartSwitchAuthenticateCallback(this, in_authenticationType, in_ids.externalId, in_ids.authenticationToken, in_extraJson, in_forceCreate, in_callback);
```

Comment:
I think this one is missing AuthName/SubType.
In C++ each of those functions define a custom callback, not this one.